### DISCUSSION TITLE
如何学习naive UI的源码

### DISCUSSION THREAD

**Discussion by guzily** [tusen-ai/naive-ui#3855] [Q&A] [answered]
我很困惑代码的目录结构和含义，src目录下为什么没有vue文件，请教一下，应该如何开始学习naive UI的源码呢，目前基本熟练了naiveUI的使用，因此想学习下源码

**Comment by 07akioni** [comment, marked as answer]
vue 文件最终也会被转换成 js，naive 直接按照转换后的格式写的。

如果你没什么基础的话只能花时间去先搞一个结构类似的最简项目，没有什么捷径。
